script.js

```diff
@@ -1,4 +1,168 @@
+// ----- Rules of game-----
+// Scissors beats paper
+// Paper beats stone
+// Stone beats scissors
+
+// User input: scissors, paper or stone
+//
+// Process:
+// - generate random number between 1 and 3
+// - correspond the number to computer's choice: scissors, paper or stone
+// - compare the user's and computer's choice to determine winner
+//
+// Output: result of game (Win, Lose or Draw)
+
+// ----------------------------------------------
+// --------------- start of code! ---------------
+// ----------------------------------------------
+
+// Create variables for formatted choices
+
+var scissorsFormatted = "Scissors ✂️";
```

Comment:
nice work formatting these putting all them into their own variables for later use